daprdocs/content/en/reference/components-reference/supported-pubsub/setup-kubemq.md

```diff
@@ -0,0 +1,110 @@
+---
+type: docs
+title: "KubeMQ"
+linkTitle: "KubeMQ"
+description: "Detailed documentation on the KubeMQ pubsub component"
+aliases:
+  - "/operations/components/setup-pubsub/supported-pubsub/setup-kubemq/"
+---
+
+## Component format
+
+To setup KubeMQ pub/sub, create a component of type `pubsub.kubemq`. See [this guide]({{< ref "howto-publish-subscribe.md#step-1-setup-the-pubsub-component" >}}) on how to create and apply a pub/sub configuration.
+
+```yaml
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: kubemq-pubsub
+spec:
+  type: pubsub.kubemq
+  version: v1
+  metadata:
+    - name: address
+      value: localhost:50000
+    - name: store
+      value: false
+```
+
+## Spec metadata fields
+
+| Field             | Required | Details                                                                                | Example                                |
+|-------------------|:--------:|----------------------------------------------------------------------------------------|----------------------------------------|
+| address           |    Y     | Address of the KubeMQ server                                                           | `"localhost:50000"`                    |
+| store             |    N     | type of pubsub, true: pubsub persisted (EventsStore), false: pubsub in-memory (Events) | `true` or `false` (default is `false`) |
+| clientID          |    N     | Name for client id connection                                                          | `sub-client-12345`                     |
+| authToken         |    N     | Auth JWT token for connection                                                          | `ew...`                                |
```

Comment:
You may want to explain the use of this Auth token some more. When is this used or can you point to documentation? The examples seems sparse. What is the format of the AuthToken and where do I get one from?

Reply:
Add KubeMQ doc reference